CMakeLists.txt

```diff
@@ -73,7 +69,11 @@ macro(targets)
   )
 endmacro()
 
-call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
+# For whatever reason when I enable this along with Connext, it fails to
+# link.  Disable for now.
+#call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
+
+targets()
```

Comment:
I think you can remove this and move the contents of the `target` macro out into the top level scope of this file again. @dirk-thomas might know better since he refactored many of the packages after the change in how rmw implementations were supported.